uber/models.py

```diff
@@ -379,6 +379,82 @@ def apply(self, params, *, bools=(), checkgroups=(), restricted=True, ignore_csr
             if not ignore_csrf:
                 check_csrf(params.get('csrf_token'))
 
+    def timespan(self, minute_increment=60):
```

Comment:
I believe @EliAndrewC 's original comment was referring to functions that were rendering HTML that were attached to the models in this file.  For instance, ```checkgroup``` was one of these.

Since then, @RobRuana has moved most of those into the ```macros``` base template alongside other functions that were able to go there.

timespan() and must_contact() are the borderline cases.  They are primarily used by templates, but technically they are just outputting formatted text, and not HTML, so they might be OK to remain here as properties of the model.

I'm fine with the current placement, and would defer to @EliAndrewC for guidance on whether we should put effort into moving them

(also, ignore my previous discussion in this comment about ```checkgroup```, it's now obsolete and was resolved by Rob)